frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -46,60 +46,84 @@ export class ImportWizard extends Wizard {
   /**
    * Edits attributes of a NICs that are being imported from source VM.
    */
-  async updateImportedNICs() {
+
+  async editNic(nic: Network) {
+    const nicDialog = new NetworkInterfaceDialog();
+    await clickKebabAction(nic.name, KEBAP_ACTION.Edit);
+    await waitForNoLoaders();
+    const networks = await nicDialog.getNetworks();
+    if (networks.length > 0) {
+      await nicDialog.selectNetwork(networks[networks.length - 1]);
+      const err = await checkForError(view.errorHelper);
+      if (err) {
+        return err;
+      }
+    } else {
+      throw Error('No available networks to assign imported NICs');
+    }
+    await click(view.confirmActionButton);
+    await waitForNoLoaders();
+    return null;
+  }
+
+  async getImportedNics() {
     const rows = await tableRows();
-    let importedNICs = rows.map((line) => {
+    return rows.map((line) => {
       const cols = line.split(/\t/);
       return {
         name: cols[networkTabCol.name],
       };
     });
+  }
+
+  async updateImportedNICs() {
+    let importedNICs = await this.getImportedNics();
     // TODO: This is horrible, but unfortunately no better way to dynamically extract only device names
     // without using ElementArrayFinder, which on the other hand may cause NoStaleElement Exceptions
     importedNICs = importedNICs.filter((_, i) => i % 3 === 0);
-
-    const NICDialog = new NetworkInterfaceDialog();
-    await asyncForEach(importedNICs, async (NIC) => {
-      await clickKebabAction(NIC.name, 'Edit');
-      await waitForNoLoaders();
-      const networks = await NICDialog.getNetworks();
-      if (networks.length > 0) {
-        await NICDialog.selectNetwork(networks[networks.length - 1]);
-      } else {
-        throw Error('No available networks to assign imported NICs');
-      }
-      await click(view.confirmActionButton);
-      await waitForNoLoaders();
+    await asyncForEach(importedNICs, async (nic) => {
+      await this.editNic(nic);
     });
   }
 
   /**
    * Edits attributes of Disks that are being imported from source VM.
    */
-  async updateImportedDisks() {
+  async editDisk(disk: Disk) {
```

Comment:
so let's call this `updateDisk` or something so we can rename the function in parent class here https://github.com/openshift/console/pull/6249/files#r485079422

Reply:
Fixed